Difi.Felles.Utility/CertificateValidator.cs

```diff
@@ -14,8 +14,48 @@ public static bool IsValidCertificate(X509Certificate2 certificate, string certi
             return ValidateCertificate(certificate, certificateOrganizationNumber).Type == CertificateValidationType.Valid;
         }
 
+        /// <summary>
+        ///     Validates the certificate and chain. Validates that certificate is
+        ///     <list type="bullet">
+        ///         <item> Not null </item>
+        ///         <item> Issued to organization number </item>
+        ///         <item> Is activated </item>
+        ///         <item> Not Expired </item>
+        ///         <item> Has a valid chain. A valid chain is one built with the allowed chain certificates.</item>
+        ///     </list>
+        /// </summary>
+        /// <param name="certificate">The certificate to validate</param>
+        /// <param name="certificateOrganizationNumber">The organization number which the certificate is issued to</param>
+        /// <param name="allowedChainCertificates"></param>
```

Comment:
Dette parameteret er ikke dokumentert for noen metoder. Er det selvforklarende, eller burde det vært noen ord om hva som forventes som input her?